### DISCUSSION TITLE
[blocks]: Image upload interface with cropper

### DISCUSSION THREAD

**Discussion by sharkaie** [shadcn-ui/ui#3302] [Blocks Request]
### Description

A input component on any connectable react-hook-form with cropper and some controls like rotation. Cropping should be fixed ratio or dynamic as per the developer configuration

Sorry i didn't find any good example.

**Comment by luco** [comment]
The component might be something like @tatwater mentioned on #1307.
Example: https://ark-ui.com/docs/components/file-upload
